### DISCUSSION TITLE
함수 컴포넌트에서 인라인 함수를 지양해야할까요?

### DISCUSSION THREAD

**Discussion by pocojang** [woowacourse/react-lotto#63] [토론해보기]
함수 컴포넌트에서 인라인 함수를 지양해야한다는 이야기가 종종 나오는데요
그 이유가 무엇일까요?

**Comment by pocojang** [comment]
- https://medium.com/steady-study/%EB%B2%88%EC%97%AD-react-%EC%9D%B8%EB%9D%BC%EC%9D%B8-%ED%95%A8%EC%88%98-%EA%B7%B8%EB%A6%AC%EA%B3%A0-%EC%84%B1%EB%8A%A5-9aef85552f2b

- https://www.reddit.com/r/reactjs/comments/d1fbiz/inline_functions_in_a_functional_component_are_no

- https://www.freecodecamp.org/news/common-mistakes-react-developers-make-and-how-to-fix-them

- https://kentcdodds.com/blog/dont-call-a-react-function-component

**Reply by zereight** [reply]
감사합니다 포코!
